code/modules/admin/verbs/randomverbs.dm

```diff
@@ -947,7 +946,12 @@ var/list/datum/outfit/custom_outfits = list() //Admin created outfits
 	to_chat(usr, "You toggled your admin antag HUD [adding_hud ? "ON" : "OFF"].")
 	message_admins("[key_name_admin(usr)] toggled their admin antag HUD [adding_hud ? "ON" : "OFF"].")
 	log_admin("[key_name(usr)] toggled their admin antag HUD [adding_hud ? "ON" : "OFF"].")
-	feedback_add_details("admin_verb","TAH") //If you are copy-pasting this, ensure the 2nd parameter is unique to the new proc!
+	if(!silent)
+		feedback_add_details("admin_verb","TAH") //If you are copy-pasting this, ensure the 2nd parameter is unique to the new proc!
+
+/client/proc/has_antag_hud()
+	var/datum/atom_hud/A = huds[ANTAG_HUD_TRAITOR]
+	return usr in A.hudusers

```

Comment:
usr is always a mob, so in this case, it's client.mob, so use that instead please.